docs/api.yaml

```diff
@@ -24,6 +24,59 @@ paths:
       responses:
         '201':
           description: Person added
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Person"
+        '400':
+          description: Missing required fields
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Missing required field(s): first_name, slack_user_id"
+        '409':
+          description: slack_user_id conflict
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Person with slack_user_id = 'W012A3CDE' already exists"
+
+  /people/{slack_user_id}:
+    get:
+      summary: Returns a specific Person by their slack_user_id with quotes in Nostalgiabot
+      parameters:
+        - in: path
+          required: true
+          name: slack_user_id
+          description: The unique ID assigned by Slack to a user
+          schema:
+            type: string
+            example: "W012A3CDE"
+      responses:
+        "200":
+          description: "Nostalgiabot remembers this person"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Person"
+        "404":
+          description: slack_user_id does not exist
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Person with slack_user_id W012A3CDE does not exist"
```

Comment:
[Nit] On the example for the `409`, you have an `=` between slack_user_id and W012A3CDE

Reply:
Do you prefer with the `=` or without it? I did this intentionally to match the actual message on this line https://github.com/revangel/nostalgiabot2/pull/33/files#diff-3e28052a1796474e35bbabaf397e4a0cR25

I can make all of the changes to make things consistent with or without the `=` though.

Reply:
Done!